packages/webhooks/lib/sync.ts

```diff
@@ -69,7 +69,7 @@ export const sendSync = async ({
             responseResults?.added === 0 && responseResults?.updated === 0 && (responseResults.deleted === 0 || responseResults.deleted === undefined);
 
         if (!webhookSettings.on_sync_completion_always && noChanges) {
-            await logCtx?.info('There were no added, updated, or deleted results. No webhook sent, as per your environment settings');
+            await logCtx?.info(`There were no added, updated, or deleted results for model ${model}. No webhook sent, as per your environment settings`);
```

Comment:
unrelated but it is confusing in the logs to see multiple times the same message when sync is multi model